README.md

```diff
@@ -39,10 +38,37 @@ You can find a list of available interfaces using
 
 `.\geneva-cli.exe list-adapters`
 
+## How to test
+
+You can test or rather verify that the intercept mode is running by simply running it with a validated strategy. The output should show that packets are being rerouted. You can monitor your adapter in [WireShark](https://www.wireshark.org/) and you should notice a huge uptick in packets being sent from your adapter.
+
+You can run `go test` to run some unit tests, currently there is only tests for validating strategies.txt
+
+## How to install as service
+
+Open PowerShell in administrator mode before running any commands
+
+### Install Command, in manual mode
+
+```New-Service -Name "Geneva-CLI" -DisplayName "WinGeneva" -Description "Geneva CLI for Windows" -StartupType Manual -BinaryPathName "Path-To-Geneva.exe --args"```
+
+### Start
+
+```Start-Service -Name "Geneva-CLI"```
```

Comment:
This did not work for me.  I ran all instructions as specified in the README in this branch.  Did I screw something up?
```
> New-Service -Name "Geneva-CLI" -DisplayName "WinGeneva" -Description "Geneva CLI for Windows" -StartupType Manual -BinaryPathName "C:\Users\Harry\Code\Lantern\geneva-cli\geneva-cli.exe intercept -interface 'vEthernet (Virtual Switch - WiFi Adapter)' -strategyFile C:\Users\Harry\Code\Lantern\geneva-cli\s.txt"

Status   Name               DisplayName
------   ----               -----------
Stopped  Geneva-CLI         WinGeneva


> Start-Service -Name "Geneva-CLI"
Start-Service : Service 'WinGeneva (Geneva-CLI)' cannot be started due to the following error:
Cannot start service Geneva-CLI on computer '.'.
At line:1 char:1
+ Start-Service -Name "Geneva-CLI"
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OpenError: (System.ServiceProcess.ServiceController:ServiceController)
    [Start-Service], ServiceCommandException
    + FullyQualifiedErrorId : CouldNotStartService,Microsoft.PowerShell.Commands.StartServiceCommand
```